frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.tsx

```diff
@@ -51,8 +53,14 @@ export const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
       direction={{ default: 'column' }}
       grow={{ default: 'grow' }}
       className="ocs-getting-started-card"
+      data-test-id={`ocs-getting-started-card-${id}`}
```

Comment:
Use space separated values so we can create selectors with `~=`

```suggestion
      data-test-id={`ocs-getting-started-card ${id}`}
```